src/mono/browser/browser.proj

```diff
@@ -341,7 +341,11 @@
       { "identity": "WasmEnableExceptionHandling",  "defaultValueInRuntimePack": "$(WasmEnableExceptionHandling)" },
       { "identity": "EnableDiagnostics",            "defaultValueInRuntimePack": "$(EnableDiagnostics)" },
       { "identity": "WasmProfilers",                "defaultValueInRuntimePack": "$(WasmProfilers)" },
-      { "identity": "EmccMaximumHeapSize",          "defaultValueInRuntimePack": "$(EmccMaximumHeapSize)" }
+      { "identity": "EmccMaximumHeapSize",          "defaultValueInRuntimePack": "$(EmccMaximumHeapSize)" },
+      { "identity": "EmccInitialHeapSize",          "defaultValueInRuntimePack": "$(EmccInitialHeapSize)" },
+      { "identity": "WasmEnableThreads",            "defaultValueInRuntimePack": "$(WasmEnableThreads)" },
```

Comment:
Why do we need relink when the switch to use threads? AFAIK it only switch between runtime packs

Reply:
because the other runtime pack have different native libs in it

Reply:
Don't we ship a default dotnet.native.wasm there as well?

Reply:
re-link is relinking dotnet.native.wasm, that's what emsdk/workload does. we ship simd/non-simd .a file in both MT/ST packs. Those .a file have different code inside based on ST or MT ifdefs.

Reply:
`PropertiesThatTriggerRelinking` trigger relink on value change. If the value is constant across builds, relink is skipped.

Reply:
OK, I'm confusing incrementalism with diff to defaults. WasmEnableThreads should not require relink (if there is no previous build). But it may require WasmBuildNative or another way to force usage of workload. Because I think that the MT pack is only part of workload ?

Regarding incrementalism we discussed with @maraf there is separate issue, which we are not trying to fix here. I stand corrected.

Reply:
So, WasmEnableThreads could be removed from here